src/colvargrid.h

```diff
@@ -1017,21 +1061,99 @@ class colvar_grid_count : public colvar_grid<size_t>
     has_data = true;
   }
 
+  /// \brief Return the average number of samples in a given "radius" around current bin
+  /// Really a hypercube of length 2*radius + 1
+  inline int local_sample_count(int radius)
```

Comment:
It would be nice to progressively move bigger functions out of colvargrid.h and into colvargrid_def.h (thus leaving the former mostly for declarations). I don't feel strongly about it if there are good reasons to do otherwise, though.

Reply:
Good point. I think the largest can even go to the cpp file, avoiding the need for an extra header. I'll give it a try.

Reply:
I'll keep only the functions I want inlined there, with the inline qualifier to make that clear.

Reply:
Some larger functions can be moved out, but that should happen in a separate PR.